scripts/linters/ts_ignore_exceptions.json

```diff
@@ -0,0 +1,145 @@
+{
+  "core/templates/domain/improvements/NeedsGuidingResponsesTaskObjectFactorySpec.ts": [
+    "task_type: '???',"
+  ],
+  "extensions/interactions/LogicProof/static/js/tools/logic-demo-test.controller.spec.ts": [
+    "properties than entries according with lint settings.",
+    "properties than {} according with lint settings."
+  ],
+  "extensions/interactions/CodeRepl/directives/code-repl-validation.service.spec.ts": [
+    "customizationArguments.language.value = 1;",
+    "customizationArguments.placeholder.value = 1;",
+    "customizationArguments.preCode.value = 1;",
+    "customizationArguments.postCode.value = 1;"
+  ],
+  "core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.controller.spec.ts": [
+    "spyOn(window, '$').withArgs('.oppia-profile-image-uploader').and",
+    ".returnValue(element);",
+    "properties than toDataURL according with lint settings."
+  ],
+  "core/templates/pages/topic-editor-page/modal-templates/questions-list-select-skill-and-difficulty-modal.controller.spec.ts": [
+    "expect(summary.isSelected).toBe(true);",
+    "expect(summary.isSelected).toBe(false);"
+  ],
+  "core/templates/mathjaxConfig.ts": ["skipStartupTypeset: true,"],
+  "core/templates/domain/collection/collection-validation.service.ts": [
+    "rid of the TS2339 error on AppConstants."
+  ],
+  "core/templates/services/request-interceptor.service.ts": [
+    "request.body = body;",
```

Comment:
There is no explanation for these ts-ignores. I think your lint checks should also require that each ts-ignore is followed by an explanation, as is best practice.

Per https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html : "It is recommended practice to have the remainder of the comment following `@ts-ignore` explain which error is being suppressed."

Reply:
added explanation